core/handle_execute_client.py

```diff
@@ -20,7 +20,7 @@ def handle_execute_client(session: Session, params: Any) -> None:
         goto_location(session, args)
     elif command_name == 'metals-show-stacktrace':
         show_stacktrace(session, args)
-    elif command_name in {'metals-doctor-run', 'metals-doctor-reload'}:
+    elif command_name in {'metals-doctor-run'}:
```

Comment:
So it's probably not a huge deal, since most people don't just sit in the doctor panel and watch it waiting for it to update, but in sublime is there no way to check if the panel (or whatever the term in sublime would be) is open? If so I think it's be better to handle these differently where run opens the doctor, but reload only "refreshes" if the doctor is already open.

Reply:
> but in sublime is there no way to check if the panel (or whatever the term in sublime would be) is open

Yes so far it isn't possible to know when the "sheet" that is used for Doctor is opened or closed. I created a separate [issue](https://github.com/scalameta/metals-sublime/issues/84) to improve the implementation once sublime API is improved.

Reply:
Gotcha, sounds good.